src/main/java/com/jw/dto/response/OrdersResponse.java

```diff
@@ -10,6 +10,7 @@
 @Getter
 @NoArgsConstructor
 @Setter
+//REVIEW-VINI: Why do we need this class? could we return a list of OrderResponse?
```

Comment:
Probably List<OrderResponse> is less complicated and more standard approach, I updated the code